main.py

```
@@ -535,6 +664,17 @@ async def activate_runtime_after_authorized(self, startup_grace: int | None = No
         if startup_grace is None:
             startup_grace = self.STARTUP_GRACE_SECONDS
 
+        # 先暖缓存，再启动抓取/发送，避免重启瞬间抢 Telethon。
```

Ruff (0.16.0)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)